README.md

```diff
@@ -176,6 +176,21 @@ manually before the backward compatibility is dropped. For every configuration u
 [the default agent config](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/agent_config.yaml)
 as a reference.
 
+### From 0.114.0 to 0.115.0
+
+- Replace the sapm exporter in the traces pipline with otlphttp.
+  
+  More details: https://github.com/signalfx/splunk-otel-collector/pull/5588.
+  
+  Configuration for `otlphttp` exporter should look like this:
+
+  ```yaml
+  otlphttp:
+    traces_endpoint: "${SPLUNK_INGEST_URL}/v2/trace/otlp"
+    headers:
+      "X-SF-Token": "${SPLUNK_ACCESS_TOKEN}"
+  ```
+
```

Comment:
2. Update `traces` pipeline to use `otlphttp` exporter instead of `sapm`:

```yaml
service:
  pipelines:
    traces:
       exporters: [otlphttp, signalfx]
```